### ISSUE TITLE
nested classes without provide

### ISSUE THREAD

**Issue by alexeagle** [angular/clutz#74]
```
/**
 * @constructor
 * @struct
 * @final
 * @implements {goog.Thenable<TYPE>}
 * @template TYPE,RESOLVER_CONTEXT
 */
goog.Promise = function(resolver, opt_context) {
/**
 * @private @final @struct @constructor
 */
goog.Promise.CallbackEntry_ = function() {}
}
```
produces a static member:
```
class Promise < TYPE , RESOLVER_CONTEXT > implements Thenable {
static CallbackEntry_ ( ) : void ;
```

**Comment by rkirov** [comment]
The static member is not output, but the type is visible because we output private members on Promise.

```
callbackEntriesTail_ : Promise.CallbackEntry_ ;
```

Surprisingly TS doesn't barf on that, but we should not output private members either.

**Comment by rkirov** [comment]
This will be fixed by #132
